Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+wd=`pwd`
+destdir="autogenerated"
+pngwidths="300 600 1200"
+totalassets=6
+
+find . -type f | grep svg >/dev/null || {
+	echo "No suitable files found. Sure we are in the right dir?"
+	return
+}
+
+# Dirty check for opened vectors
+grep -l -E ' d="[^"]*[^zZ]"' *.svg >"$destdir/.temp" &&  (
+	echo "This/those files seems to have open paths!";
+	cat  "$destdir/.temp"
+	echo
```

Comment:
If we keep this check here:

- Use `grep -q` instead of creating a temporary file
- Exit with an error status (`exit 1`) when this condition occurs

Reply:
Done.